remail/email_api/service.py

```
@@ -0,0 +1,488 @@
+        password = input("Gebe das IMAPpasswort ein, um es auf deinem Rechner zu hinterlegen: ")
+        keyring.set_password("remail/IMAP","thatchmilo35@gmail.com",password)
+
+def change_credentials_exchange():
+    password = input("Gebe das Exchangepasswort ein, um es auf deinem Rechner zu hinterlegen: ")
+    keyring.set_password("remail/exchange","praxisprojekt-remail@uni-due.de",password)
+def change_credentials_imap():
+    password = input("Gebe das IMAPpasswort ein, um es auf deinem Rechner zu hinterlegen: ")
+    keyring.set_password("remail/IMAP","thatchmilo35@gmail.com",password)
+
+def safe_file(filename:str,content:bytes)->str:
+    if len(content) > max_size:
```

Ruff (0.8.0)

Undefined name `max_size`

(F821)